helm/ph-ee-engine/values.yaml

```diff
@@ -1551,6 +1552,7 @@ connector_bulk:
     deduplication:
       enabled: true  
   aws:
+    host: "<host>"
     region: "<region>"
```

Comment:
configure minio default  credentials here and remove these references from g2pSandbox chart

Reply:
done